yarn.lock

```diff
@@ -3919,7 +3919,7 @@
   resolved "https://registry.npmjs.org/@types/keygrip/-/keygrip-1.0.1.tgz#ff540462d2fb4d0a88441ceaf27d287b01c3d878"
   integrity sha1-/1QEYtL7TQqIRBzq8n0oewHD2Hg=
 
-"@types/lodash@^4.14.138", "@types/lodash@^4.14.139", "@types/lodash@^4.14.149":
+"@types/lodash@^4.14.138", "@types/lodash@^4.14.139":
```

Comment:
Are these changes related to this PR?

Reply:
I do not understand why this is here now. I only ran `yarn`